HappyCamps/src/app/home/home.component.html

```diff
@@ -0,0 +1,2 @@
+<h1> Hello DashBoad</h1>
```

Comment:
Can delete this hello DashBoard, it's pointless

Reply:
Done